### PR TITLE
Add more input sanitization

### PR DESCRIPTION
**Pull request by wolframroesler**
Fixes additional crashes found by the AFL fuzz tester.

### CODE REVIEW

File: libmdnsd/1035.c [hunk 1]
```diff
@@ -249,7 +256,8 @@ static int _rrparse(struct message *m, struct resource *rr, int count, unsigned
 	int i;
 
 	for (i = 0; i < count; i++) {
-		_label(m, bufp, &(rr[i].name));
+		if (_label(m, bufp, &(rr[i].name))
```
Comment (troglobit) [inline libmdnsd/1035.c:259]: Seem to be missing an ending `)` here.   Travis-CI fails to build ...

Comment (wolframroesler) [reply libmdnsd/1035.c]: How embarrassing. Fixed.

**Comment by troglobit** [comment]
Looks good, thanks!

Just FYI, I'm holding off the next release pending your fixes. Just drop me a note, email, smoke signals, when you're done and I can prep the next release :)

**Comment by wolframroesler** [comment]
Parallel fuzzing executed my test program over a billion times and didn't find any more crashes or hangs.

```
status check tool for afl-fuzz by <lcamtuf@google.com>

/home/wolfram/sourceSDK/aflbuild/findings
Summary stats
=============

       Fuzzers alive : 256
      Total run time : 1210 days, 11 hours
         Total execs : 1243 million
    Cumulative speed : 2904 execs/sec
       Pending paths : 99 faves, 124183 total
  Pending per fuzzer : 0 faves, 485 total (on average)
       Crashes found : 0 locally unique
```

So, it seems we're done here.

BTW, in case you're wondering, I'm doing this on behalf of https://github.com/devolo.

**Comment by troglobit** [comment]
Very cool, thanks for the update! :)

I'll start preparing a release then.

Thanks, I was a bit curious who it was that found some business value in mdnsd.  Give them my best regards! :)

**Comment by troglobit** [comment]
There, v0.10 out now.  Thank you so much to you and the rest of the team, again!  In case you're going with a patched version, you might want to import 35ba357 to handle a minor regression I found yesterday.

**Pull request merged**